src/scene/scene.js

```diff
@@ -116,6 +120,12 @@ function Scene() {
     this._skyboxIntensity = 1;
     this._skyboxMip = 0;
 
+    this._skyboxRotation = new Quat();
+    this._skyboxRotationMatrix = new Mat3();
```

Comment:
looks like this can be initialised to null?

Reply:
I was treating this _skyboxRotation more like a value than an object - which lines up with the code in: `Scene.prototype.applySettings()` - I can change it but is it better to do a new object allocation in `Scene.prototype.applySettings()`?

Reply:
After more thought I do prefer it be treated like a value vs object in this case - I did remove the 3X3 matrix allocation though

Reply:
Oh I see there are two matrices. I would say you should handle Matrix and Matrix4 similarly - either initialise both on construction or both on demand. (Unless the two matrices are different in some way I haven't noticed).